utilities/write_batch_with_index/write_batch_with_index_internal.cc

```diff
@@ -94,9 +94,11 @@ int WriteBatchEntryComparator::operator()(
     return -1;
```

Comment:
Would be great to add inline comments to the definition of operator() that how entries are compared in presence and absence of search keys.